src/hooks/use-section-toggle/useSectionToggle.js

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react';
+
+import { expandAllFunction } from '@folio/stripes/components';
+
+export default function (initialSections) {
```

Comment:
I think that it will be better to set name for this hook here. Searching by the code could be easier in future.